src/World.cpp

```diff
@@ -544,7 +544,7 @@ void cWorld::Start(void)
 	if (GetDimension() == dimOverworld)
 	{
 		AString MyNetherName = GetName() + "_nether";
-		AString MyEndName = GetName() + "_end";
+		AString MyEndName = GetName() + "_the_end";
```

Comment:
again, I think we should check for both `_the_end` and `_end`. Change the block below to:

```
 		if (cRoot::Get()->GetWorld(MyEndName) == nullptr)
 		{
 			MyEndName = GetName() + "_end";
 			if (cRoot::Get()->GetWorld(MyEndName) == nullptr)
 			MyEndName = "";
 		}

```